### PR TITLE
Update pad.py

### PR DESCRIPTION
**Pull request by tjingrant**
Compare https://docs.scipy.org/doc/numpy-1.13.0/reference/generated/numpy.pad.html with https://github.com/onnx/onnx/blob/master/docs/Operators.md#Pad, numpy pad_width is the transpose of onnx pads.

### CODE REVIEW

**Comment by tjingrant** [comment]
related to https://github.com/onnx/onnx/issues/261.
We've fixed the pads param but the test is wrong.

File: docs/Operators.md [hunk 1]
```diff
@@ -2475,7 +2475,7 @@ node = onnx.helper.make_node(
     outputs=['y'],
     mode='constant',
     value=1.2,
-    pads=[0, 0, 0, 0, 1, 2, 3, 4],
+    pads=[0, 1, 0, 2, 0, 3, 0, 4],
```
Comment (bddppq) [inline docs/Operators.md]: This should be [0, 0, 1, 3, 0, 0, 2, 4]

**Comment by bddppq** [comment]
@onnxbot test this please

**Comment by bddppq** [comment]
Thanks!

**Comment by tjingrant** [comment]
My bad, wrong direction of the transposition.

**Pull request merged**